pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -44,11 +49,26 @@ func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *
 	// Do not fail, but continue after logging error as we can still dump flows with no timestamps.
 	sysctl.EnsureSysctlNetValue("netfilter/nf_conntrack_timestamp", 1)
 
-	return &connTrackSystem{
+	ct := &connTrackSystem{
```

Comment:
You don't seem to be using this variable. Why not keep the same return that is previously there?

Reply:
Thanks, done.